src/Categories/Adjoint/Parametric.agda

```diff
@@ -0,0 +1,150 @@
+module Categories.Adjoint.Parametric where
+
+open import Level
+
+open import Data.Product
+open import Categories.Category.Core using (Category)
+open import Categories.Functor using (Functor; _∘F_) renaming (id to idF)
+open Categories.Functor.Functor using (F₀; F₁; homomorphism; identity; F-resp-≈)
+open import Categories.Adjoint
+open import Categories.Category.Construction.Functors
+open import Categories.Diagram.Cowedge
+open import Categories.NaturalTransformation.Dinatural
+open import Categories.Functor.Bifunctor
+open import Categories.Morphism.Properties
+open import Categories.Functor.Bifunctor.Properties
+open import Categories.NaturalTransformation
+open import Categories.NaturalTransformation.NaturalIsomorphism
+open import Categories.Morphism.Reasoning as MorphismReasoning
+import Categories.Morphism.Reasoning as MR
+open Categories.NaturalTransformation.NaturalTransformation using (η)
+open import Function.Equality using (_⟨$⟩_; _⟶_; cong; _∘_)
+
+open import Categories.Adjoint.Equivalents
+
+private
+  variable
+    o ℓ e : Level
+    C D E : Category o ℓ e
```

Comment:
No point creating generalized variables here, only to immediately shadow them in the next definition...